src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -137,6 +137,7 @@ export class ClearAllOutputsAction extends TooltipFromLabelAction {
 
 	public override async run(context: URI): Promise<void> {
 		const editor = this._notebookService.findNotebookEditor(context);
+		editor.model.sendNotebookTelemetryActionEvent(TelemetryKeys.NbTelemetryAction.ClearResults);
```

Comment:
By default, do we not get any telemetry by default from actions?

Reply:
All actions are already logged, so unless you're adding additional metadata you shouldn't need to do this. Although there's something to be said for consolidating all the NB actions under a single bucket so they are easier to correlate.

Reply:
I'll remove the events without any metadata for now.